roofit/roofit/src/RooChebychev.cxx

```diff
@@ -236,6 +237,30 @@ RooSpan<double> RooChebychev::evaluateBatch(std::size_t begin, std::size_t batch
 }
 ////////////////////////////////////////////////////////////////////////////////
 
+RooSpan<double> RooChebychev::evaluateSpan(BatchHelpers::RunContext& evalData, const RooArgSet* normSet) const {
+
+  RooSpan<const double> xData = _x->getValues(evalData, normSet);
+  if (xData.empty()) {
```

Comment:
Is this needed?

Reply:
Not needed, see above. I will either be size 1 or the size of the dataset.